build/buildconstants/params_2k.go

```diff
@@ -169,7 +169,7 @@ func init() {
 	UpgradeWaffleHeight = getUpgradeHeight("LOTUS_WAFFLE_HEIGHT", UpgradeWaffleHeight)
 	UpgradePhoenixHeight = getUpgradeHeight("LOTUS_PHOENIX_HEIGHT", UpgradePhoenixHeight)
 	UpgradeTuktukHeight = getUpgradeHeight("LOTUS_TUKTUK_HEIGHT", UpgradeTuktukHeight)
-	UpgradeXxHeight = getUpgradeHeight("LOTUS_XX_HEIGHT", UpgradeXxHeight)
+	UpgradeTeepHeight = getUpgradeHeight("LOTUS_XX_HEIGHT", UpgradeTeepHeight)
```

Comment:
Just confirming this should be `LOTUS_XX_HEIGHT` instead of `LOTUS_TEEP_HEIGHT`

Reply:
```suggestion
	UpgradeTeepHeight = getUpgradeHeight("LOTUS_TEEP_HEIGHT", UpgradeTeepHeight)
```
good catch, it should be set, in fact I'm not sure we should have even done the `XX` form of this.